### PR TITLE
OVC fix

### PR DESCRIPTION
**Pull request by ahmedelsayed-93**

### CODE REVIEW

File: functional_testing/Openvcloud/utils/utils.py [hunk 1]
```diff
@@ -407,18 +404,19 @@ def get_machine_nodeID(self, machineId):
         machine = ccl.vmachine.get(machineId)
         stackID = machine.stackId
         nodeID = ccl.stack.get(stackID).referenceId
-        return nodeID
+        return int(nodeID)
```
Comment (john-kheir) [inline functional_testing/Openvcloud/utils/utils.py]: check that there is nothing broken or u can remove the int ant put inside the function

File: functional_testing/Openvcloud/utils/utils.py [hunk 2]
```diff
@@ -407,18 +404,19 @@ def get_machine_nodeID(self, machineId):
...
-                return node.id
-        return -1
+        stacks = ccl.stack.list()
+        for stackId in stacks:
+            nodeId = int(ccl.stack.get(stackId).referenceId)
+            node = scl.node.get(int(nodeId))
+            if node.active and 'fw' in node.roles and nodeId != current_VFW_nodeId:
+                return nodeId
+        else:
+            return False
```
Comment (john-kheir) [inline functional_testing/Openvcloud/utils/utils.py]: Change this False to -1 or check if it is there in the ACL

Comment (ahmedelsayed-93) [reply functional_testing/Openvcloud/utils/utils.py]: only used in the implemented testcase in this PR

File: functional_testing/Openvcloud/utils/utils.py (hunk 1 above)
Comment (ahmedelsayed-93) [reply functional_testing/Openvcloud/utils/utils.py]: only used one time in this ```ovc_master_hosted/OVC/a_basic/machine_tests.pytestcases test007_cleanup_vxlans_for_stopped_deleted_vms```

**Pull request merged**